src/Plugin/Visualisation/Style/AxisChart.php

```diff
@@ -685,6 +685,20 @@ public function build() {
         '#type' => 'container',
         '#attributes' => ['class' => ['table-chart--actions']],
       ];
+
+      $file_uri = $this->getDatasetDownloadUri($this->visualisation->getEntity());
```

Comment:
Noting that is only on an Axis chart, what about a visualisation that is not an axis chart, eg table or future vis.

feels like `getDatasetDownloadUri` should be abstracted to parent class and not specifically on axis only